images/egress/dns-proxy/egress-dns-proxy.sh

```diff
@@ -58,6 +58,11 @@ defaults
     timeout server          1m
     timeout http-keep-alive 100s
     timeout check           10s
+
+    # Address resolution methods
+    # Previous address found in the state file (if any) is used first,
+    # otherwise the libc's resolver is used.
+    default-server init-addr last,libc
```

Comment:
`last,libc` is the default value of that option, so you don't even need to set it...